example/.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": ["@zendeskgarden", "@zendeskgarden/eslint-config/plugins/notice", "prettier"],
+  "parserOptions": {
+    "requireConfigFile": false
+  },
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx"],
+      "extends": ["@zendeskgarden/eslint-config/plugins/typescript", "prettier"]
+    }
+  ]
+}
```

Comment:
It may be worthwhile to move eslint to the root of the project instead of have it nested individually.

Reply:
Can we capture this as a follow-on?